### PR TITLE
Bugfixes in simple-client-ts.ts and simple-client.js

### PR DESCRIPTION
**Pull request by pirminkager**
- added require("util")
    - namespace array fix

### CODE REVIEW

File: packages/node-opcua-samples/bin/simple_client.js [hunk 1]
```diff
@@ -444,7 +445,7 @@ async function main() {
     console.log(" --- NAMESPACE ARRAY ---");
     const namespaceArray = dataValue.value.value;
     for (const namespace of namespaceArray) {
-        console.log(" Namespace ", namespace.index, "  : ", namespace);
+        console.log(" Namespace ", namespaceArray.indexOf(namespace), "  : ", namespace);
```
Comment (erossignon) [inline packages/node-opcua-samples/bin/simple_client.js:448]: Please revert, the intend is really  `namespace.index`

File: packages/node-opcua-samples/bin/simple_client_ts.ts [hunk 2]
```diff
@@ -455,7 +455,7 @@ async function main() {
     console.log(" --- NAMESPACE ARRAY ---");
     const namespaceArray = dataValue.value.value;
     for (const namespace of namespaceArray) {
-        console.log(" Namespace ", namespace.index, "  : ", namespace);
+        console.log(" Namespace ", namespaceArray.indexOf(namespace), "  : ", namespace);
```
Comment (erossignon) [inline packages/node-opcua-samples/bin/simple_client_ts.ts:458]: Please revert, the intend is really  `namespace.index`

Comment (pirminkager) [reply packages/node-opcua-samples/bin/simple_client_ts.ts:458]: I am getting `undefined` when i call `namespace.index` but i would expect the Indexnumber.
Why could that be?

Comment (erossignon) [reply packages/node-opcua-samples/bin/simple_client_ts.ts:458]: sorry,I confirmed ,  you are right.
 namespaceArray returns a bunch of string not a array of Namespace Object.

**Pull request closed without merging**